proxy.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * @internal Symbol used to track if the object has been mutated
+ */
+export const _MUTATED = Symbol("mutated");
+
+/**
+ * @internal Symbol used to track if the object is being tracked
+ */
+export const _TRACKED = Symbol("tracked");
```

Comment:
Can we leave them internal, or is there a good reason to export them?
```suggestion
/**
 * @internal Symbol used to track if the object has been mutated
 */
const _MUTATED = Symbol("mutated");

/**
 * @internal Symbol used to track if the object is being tracked
 */
const _TRACKED = Symbol("tracked");
```

Reply:
I think we use them for testing.
I don't know if there's another way of exposing these to tests but not to the public API.

Reply:
I think we can use `Symbol.for` then

Reply:
Or just make sure we only export things internally but don't leak it to package consumers :)